src/node.h

```diff
@@ -409,17 +409,23 @@ extern "C" NODE_EXTERN void node_module_register(void* mod);
 # define NODE_MODULE_EXPORT __attribute__((visibility("default")))
 #endif
 
+#ifdef NODE_SHARED_MODE
+# define NODE_CTOR_PREFIX
+#else
+# define NODE_CTOR_PREFIX static
+#endif
```

Comment:
@bnoordhuis Correct, here is an example  (notice uppercase T for export): 

```
nm -a out/Release/lib.target/libnode.so.48 | grep _register_
...
0000000000622d90 T _register_async_wrap
0000000000622e10 T _register_buffer
0000000000622dd0 T _register_cares_wrap
0000000000622e30 T _register_config
0000000000622e50 T _register_contextify
0000000000623070 T _register_crypto
0000000000622e70 T _register_fs
0000000000622db0 T _register_fs_event_wrap
0000000000622e90 T _register_http_parser
0000000000622df0 T _register_js_stream
0000000000622eb0 T _register_os
0000000000622f30 T _register_pipe_wrap
0000000000623010 T _register_process_wrap
0000000000622f50 T _register_signal_wrap
0000000000622f70 T _register_spawn_sync
0000000000622f90 T _register_stream_wrap
0000000000622fb0 T _register_tcp_wrap
0000000000622fd0 T _register_timer_wrap
00000000006230d0 T _register_tls_wrap
0000000000622ff0 T _register_tty_wrap
0000000000623030 T _register_udp_wrap
0000000000622ed0 T _register_util
0000000000623050 T _register_uv
0000000000622ef0 T _register_v8
0000000000622f10 T _register_zlib
```

This change was in the original PR by Fedor. I believe he included it due to Electron, see here https://github.com/electron/node/commit/0828dfa193038209b7b337ba5479c3373f7cf87f and the uses here https://github.com/electron/electron/commit/69adff19fe610800de900fc9358a62a402769bac